symplyphysics/symbols/classical_mechanics.py

```diff
@@ -47,6 +52,12 @@
 **Length** is a measure of a size of an object.
 """
 
+thickness = SymbolNew("d", units.length)
+"""
+**Thickness** is a measure of a size of an object, usually the separation between two layers, or
```

Comment:
It's pretty much the description of `distance`.

Reply:
The question is are they interchangeable? "The distance of the box is 10 cm" and "the thickness of the two points is 3 km" do not make sense.

Reply:
Ok. What about `length`? Length, width and height are pretty much interchangeable.

Reply:
We can have laws that feature all three parameters at once. [Here](https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/kinematics/rotational_inertia/geometries/slab_about_perpendicular_axis_through_center.py) I imagine we can and need to use both `length` and `width` since we need to describe two dimensions of the body. We can require the user to clone and rename them as needed but it seems reasonable to me to let them all be here. Almost always `length` is used to talk about the general and biggest dimension of the object, and if it's already in use, `width` is used as the secondary and smaller dimension. `height` is most commonly the dimension along the `z`-axis. `thickness`, on the other hand, is specifically about the separation between layers or similar things, which I mentioned in the text.

Reply:
I think `thickness` could be understood in two ways:
1. Size of a layer of something, i.e. thickness of asphalt.
2. Separation of two layers of something.

1 is interchangeable with `width` or `height`, depending on position, vertical or horizontal.
2 is interchangeable with `distance` but requires rephrasing, i.e. `thickness of air gap` = `distance between walls of the gap`

I suggest to avoid using 2nd meaning of the thickness and use `distance` instead (with a bit of rephrasing). And use `width` , `height` or `length` whenever we use 1st meaning of the thickness (I doubt we have any).